pkg/instrumentation/podmutator_test.go

```diff
@@ -1034,58 +1031,31 @@ func TestMutatePod(t *testing.T) {
 			},
 		},
 		{
-			name: "dotnet injection feature gate disabled",
+			name: "apache httpd injection, true",
```

Comment:
Maybe a bad merge in this file?  Looks like some existing tests were removed instead of this test being added.

Reply:
Oh sorry - bad merge indeed. Fixed.